### DISCUSSION TITLE
How do I update the app?

### DISCUSSION THREAD

**Discussion by burritocat23** [bttv-android/bttv#609] [Q&A] [answered]
I downloaded the .apk file and when attempting to sign in it tells me to update the app. Is there an update file I can install or...? I'm not knowledgeable on coding as much as I should be so a solution could be staring me right in the face and I'm just missing it. Thanks in advance.

**Comment by FoseFx** [comment, marked as answer]
When a new version is available you should see a purple-ish banner on the bottom when starting the app. You can simply tap on it and it will download and install the new version.
**Note**: The first time it will ask you to allow bttv-android to be a source for new applications. You should have seen a similar popup when you first install the app (but for your browser then).

The app will also periodically check for updates in the background and give you a notification, you can also tap on that to start the update procedure. You can disable this in the bttv-android settings.

You can always manually check for updates by going to the bttv-android settings and tap on "Check for updates now"

**Reply by burritocat23** [reply]
None of this ever showed up for me. No banner, nothing was asked after I downloaded it, nothing. After I download the app I'm simply given the option to login or sign up and after attempting to login it tells me "Please update your app to continue", but I'm not given a way to do it. I also don't see an option to update from my Android settings either.

**Reply by burritocat23** [reply]
I figured out what the problem was. I downloaded an old version (25.1 to be specific). I guess I should've double checked to see if there were any new versions out. My bad!

**Comment by AllNattyGymster** [comment]
How can i go back to 0.41 .. i updated and i don't like it. I was able to press the button on my wired headphones to pause and start the music. With 0.43 when i press the button now on the headphones it doesn't restart my music and i have to re-open my phone and hit play manually :(
